datastore/data-loader/client_knmi_station.py

```diff
@@ -29,6 +29,32 @@
 regex_time_period = re.compile(r"(\d+) (Hours|Min)", re.IGNORECASE)
 
 
+def convert_standard_names_to_cf(standard_name):
+    standard_name_mapping = {
+        "cloud_cover": "cloud_area_fraction",
+        "total_downwelling_shortwave_flux_in_air": "surface_downwelling_shortwave_flux_in_air",
+        "precipitation_rate": "rainfall_rate",
+        "air_pressure_at_sea_level": "air_pressure_at_mean_sea_level",
+    }
+    return standard_name_mapping.get(standard_name, standard_name)
+
+
+# NOTE: Only units are converted currently, not values.
+def convert_unit_names(unit):
+    unit_mapping = {
+        "degrees Celsius": "degC",
+        "ft": "m",
+        "min": "s",
+        "degree": "degrees",
+        "%": "percent",
+        "mm": "kg/m2",
+        "m s-1": "m/s",
+        "octa": "oktas",
+        "W m-2": "W/m2",
```

Comment:
These are the one that the ingestor translates units to?

Reply:
Yes, that way both of the `data-loaders` have the same `"output"` and we can use the same test comparison data.